tools/start_server.py

```diff
@@ -263,14 +269,59 @@ def isFileNameValid(_file_name):
       "success": True
     }
 
+  def handle_start_video_stream(self, _method, _params):
+    if 'url' not in _params:
+      return self.gen_error_msg('Missing mandatory param \'url\'')
+
+    if 'config' not in _params:
+      return self.gen_error_msg('Missing mandatory param \'config\'')
+
+    if _params['config']['protocol'] == 'RTP':
```

Comment:
I would recommend to use `get()` instead of direct map access because all config parameters are not mandatory. Python will throw the exception if you try to access key which does not exist

Reply:
good idea, non mandatory access replaced with get in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/ba8ca359b09050171923c8073e10909db86dd9e6